pkg/antctl/command_definition.go

```diff
@@ -38,9 +38,10 @@ import (
 type formatterType string
 
 const (
-	jsonFormatter  formatterType = "json"
-	yamlFormatter  formatterType = "yaml"
-	tableFormatter formatterType = "table"
+	jsonFormatter      formatterType = "json"
+	yamlFormatter      formatterType = "yaml"
+	tableFormatter     formatterType = "table"
+	rawStringFormatter formatterType = "rawString"
```

Comment:
I think it would be more user-friendly to name it `raw` or `rawstring`

Reply:
Changed to `raw`